test/abi.jl

```diff
@@ -82,6 +82,9 @@ using Test
     @test dx[] ≈ 3.0
     @test dy[] ≈ 2.0
 
+    mid, = Enzyme.autodiff((f, x) -> f[1](x), [x->x*x], Active(2.0))
```

Comment:
```suggestion
    mid, = Enzyme.autodiff((fs, x) -> fs[1](x), (x->x*x,), Active(2.0))
```

Since the moment you have more than one entry in `fs` the array version will explode